sdk/communication/azure-communication-administration/README.md

```diff
@@ -118,44 +118,44 @@ also takes in a list of communication token scopes. Scope options include:
 
 <!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L93-L96 -->
 ```java
-List<String> scopes = new ArrayList<>(Arrays.asList("chat"));
 CommunicationUserToken userToken = communicationIdentityClient.issueToken(user, scopes);
 System.out.println("Token: " + userToken.getToken());
 System.out.println("Expires On: " + userToken.getExpiresOn());
+return userToken;
 ```
 
 ### Revoking all tokens for an existing user
 Use the `revokeTokens` function to revoke all the issued tokens of a user.
 
 <!-- embedme ./src/samples/java/com/azure/communication/administration/ReadmeSamples.java#L108-L109 -->
 ```java
-// revoke tokens issued for the user prior to now
```

Comment:
Why this change? The existing snippet seemed fine

Reply:
same as the issue above